pkg/scheduler/core/generic_scheduler.go

```diff
@@ -412,6 +399,11 @@ func (g *genericScheduler) numFeasibleNodesToFind(numAllNodes int32) (numNodes i
 // Filters the nodes to find the ones that fit the pod based on the framework
 // filter plugins and filter extenders.
 func (g *genericScheduler) findNodesThatFitPod(ctx context.Context, prof *profile.Profile, state *framework.CycleState, pod *v1.Pod) ([]*v1.Node, framework.NodeToStatusMap, error) {
+	s := prof.RunPreFilterPlugins(ctx, state, pod)
```

Comment:
nit: it seems `state` doesn't need to be passed in? as there is no place instantiating earlier than RunPreFilterPlugins?

Reply:
we need to pass it, this is where the prefilters will write the precomputed state.

Reply:
sg